symphony/bdk/core/service_factory.py

```diff
@@ -119,12 +121,25 @@ def get_datafeed_loop(self) -> AbstractDatafeedLoop:
             self._config
         )
 
-    def get_health_service(self):
+    def get_health_service(self) -> HealthService:
         """Returns a fully initialized HealthService
 
         :return: a new HealthService instance
         """
-        return HealthService(AgentSystemApi(self._agent_client), SignalsApi(self._agent_client))
+        return HealthService(
```

Comment:
do we have different formatting config or was it not formatted before?

Reply:
It was not formatted before.